layouts/research_lines/single.html

```diff
@@ -0,0 +1,111 @@
+{{- define "main" -}}
+
+<article class="article article-project">
+
+  <div class="warp" style="background-color: #f5f5f5;">
```

Comment:
We should probably extract the color to the custom SCSS file

Reply:
This should be resolved now (before previous meeting)